crypto/src/tests/secp256k1_tests.rs

```diff
@@ -258,11 +258,11 @@ proptest::proptest! {
 
         // same recovered pubkey are recovered
         let recovered_key = signature.sig.recover(&hashed_msg).unwrap();
-        let recovered_key_1 = signature_1.recover_verifying_key(message).expect("couldn't recover pubkey");
+        let recovered_key_1 = signature_1.recover_verify_key(message).expect("couldn't recover pubkey");
```

Comment:
it seems that some tests fail because of the renaming

Reply:
Yep, this is reverted!